source/technical/networks/networks_stitching.rst

```diff
@@ -22,15 +22,19 @@ In the following sections, this workflow is described for different settings.
 Configuring a Stitchable Network
 ________________________________
 
-Follow the technical documentation for :ref:`network-cli-create` using the CLI, but replace the provider network with the appropriate external testbed (e.g. replace ``physnet1`` with ``exogeni``). 
-In this documentation, we will describe how to stitch to the ExoGENI testbed:
+In this documentation, we will describe how to stitch to the ExoGENI testbed.
+Your first step will require creating a stitchable network. Unlike creating
+other networks on Chameleon, stitchable networks can only be created by first
+reserving a stitchable VLAN segment using the CLI (See
+:ref:`reservation-cli-vlan`). Once you reserve any VLAN segment, your network
+will be created automatically. To reserve a segment on the appropriate
+external testbed make sure to include ``exogeni`` as the ``phyical_network``
+in the ``resource_properties`` attribute. An example is provided below:
 
 .. code-block:: bash
 
-   openstack network create --provider-network-type vlan --provider-physical-network exogeni <network_name>
+   blazar lease-create --reservation resource_type=network,network_name="my-stitchable-network",resource_properties='["==","$physical_network","exogeni"]' --start-date "2015-06-17 16:00" --end-date "2015-06-17 18:00" my-stitchable-network-lease
```

Comment:
```suggestion
   blazar lease-create --reservation resource_type=network,network_name=my-stitchable-network,resource_properties='["==","$physical_network","exogeni"]' --start-date "2015-06-17 16:00" --end-date "2015-06-17 18:00" my-stitchable-network-lease
```

I noticed if network_name is in quotes, the network name on openstack has quotes in it, which is not desirable.